src/Microsoft.Identity.Client/ClientApplicationBase.WithBuilders.cs

```diff
@@ -35,13 +35,81 @@
     public abstract partial class ClientApplicationBase : IClientApplicationBaseExecutor
     {
+        /// <summary>
+        /// [V3 API] Attempts to acquire an access token for the <paramref name="account"/> from the user token cache, 
+        /// with advanced parameters controlling the network call. See https://aka.ms/msal-net-acquiretokensilent for more details
+        /// </summary>
+        /// <param name="scopes">Scopes requested to access a protected API</param>
+        /// <param name="account">Account for which the token is requested. This parameter is optional.
+        /// If nothing is passed and no Account or LoginHint are provided then if one and only
+        /// one account is in the cache, that account is used.  Otherwise, an exception will be thrown.  <see cref="IAccount"/></param>
+        /// <returns>An <see cref="AcquireTokenSilentParameterBuilder"/> used to build the token request, adding optional
+        /// parameters</returns>
+        /// <exception cref="MsalUiRequiredException">will be thrown in the case where an interaction is required with the end user of the application,
+        /// for instance, if no refresh token was in the cache,a or the user needs to consent, or re-sign-in (for instance if the password expired),
+        /// or the user needs to perform two factor authentication</exception>
+        /// <remarks>
+        /// The access token is considered a match if it contains <b>at least</b> all the requested scopes. This means that an access token with more scopes than
+        /// requested could be returned as well. If the access token is expired or close to expiration (within a 5 minute window),
+        /// then the cached refresh token (if available) is used to acquire a new access token by making a silent network call.
+        ///
+        /// See also the additional parameters that you can set chain:
+        /// <see cref="AbstractAcquireTokenParameterBuilder{T}.WithAuthority(string, bool)"/> or one of its
+        /// overrides to request a token for a different authority than the one set at the application construction
+        /// <see cref="AcquireTokenSilentParameterBuilder.WithForceRefresh(bool)"/> to bypass the user token cache and
+        /// force refreshing the token, as well as
+        /// <see cref="AbstractAcquireTokenParameterBuilder{T}.WithExtraQueryParameters(Dictionary{string, string})"/> to
+        /// specify extra query parameters
+        /// 
+        /// </remarks>
+        public AcquireTokenSilentParameterBuilder AcquireTokenSilent(IEnumerable<string> scopes, IAccount account)
+        {
+            return AcquireTokenSilentParameterBuilder.Create(this, scopes, account);
+        }
+
+        /// <summary>
+        /// [V3 API] Attempts to acquire an access token for the <paramref name="loginHint"/> from the user token cache, 
+        /// with advanced parameters controlling the network call. See https://aka.ms/msal-net-acquiretokensilent for more details
```

Comment:
>with advanced parameters controlling the network call [](start = 12, length = 53)

what does this mean? which params and what control?

Reply:
Oh I copied the comments from the overload, sorry I didn't point that out. I reviewed it and removed that sentence, it isn't saying anything.